lib/response_bank.rb

```diff
@@ -25,8 +25,8 @@ def write_to_backing_cache_store(_env, key, payload, raw:, expires_in: nil)
       cache_store.write(key, payload, raw: raw, expires_in: expires_in)
     end
 
-    def read_from_backing_cache_store(_env, cache_key, backing_cache_store: cache_store)
-      backing_cache_store.read(cache_key)
+    def read_from_backing_cache_store(_env, cache_key, raw:, backing_cache_store: cache_store)
+      backing_cache_store.read(cache_key, raw: raw)
```

Comment:
Since `write_to_backing_cache_store` and `read_from_backing_cache_store` only receive `raw: true`, shouldn't we remove the option and just write/read raw from ResponseBank directly?